product-service/products.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "count": 4,
+    "description": "Улучшение проекта существующего кода» пролили свет на процесс рефакторинга, описав принципы и лучшие приёмы его осуществления.",
```

Comment:
Let's use English because it will be used for EN auditory too

Reply:
Fixed in 3ad0de7